pallets/Polkadex_Ido/src/test.rs

```diff
@@ -167,86 +162,62 @@ fn test_whitelist_investor() {
 fn test_participate_in_round() {
     let balance: Balance = 100;
     let investor_address: u64 = 4;
-    let round_id = create_hash_data(&1u32);
-    let open_block_number = 11;
-    let closing_block_number = 50;
+    let funding_period = 10;
+    let amount: Balance = 200;
+    let min_allocation: Balance = 100;
+    let max_allocation: Balance = 400;
     let cid = [0_u8;32].to_vec();
     ExtBuilder::default().build().execute_with(|| {
-        assert_noop!(
-            PolkadexIdo::participate_in_round(Origin::signed(ALICE.clone()), round_id, balance),
-            Error::<Test>::NotWhiteListed
-        );
 
         assert_eq!(
             PolkadexIdo::register_round(
-                Origin::signed(ALICE.clone()),
+                Origin::signed(ALICE),
                 cid,
                 AssetId::CHAINSAFE(H160::from_low_u64_be(24)),
```

Comment:
Did I not break this last week?

Reply:
I am branching from develop which is not updated to the lastest change from the main poldexprimitives repo